.gitignore

```diff
@@ -9,3 +9,5 @@
 /gtk/gschemas.compiled
 /tools/regtool
 /web/oscmix.wasm
+/coremidiio
+.DS_Store
```

Comment:
I think this is best done in your personal `~/.config/git/ignore` file, since `.DS_Store` is not actually a build artifact of oscmix, but created by Finder.

That way, it will also be ignored on other repositories.